mixer/adapter/list/list.go

```diff
@@ -79,7 +80,28 @@ func (h *handler) HandleListEntry(_ context.Context, entry *listentry.Instance)
 		return adapter.CheckResult{}, err
 	}
 
-	found, err := l.checkList(entry.Value)
+	var value string
+	isString := true
+
+	switch entryVal := entry.Value.(type) {
+	case *v1beta1.Value:
+		if entryVal.GetStringValue() != "" {
```

Comment:
It'd be more efficient to call GetStringValue once and cache the results. Same for GetIpAddressValue below

Reply:
Fixed